plugins/modules/identity/ipa/ipa_sudorule.py

```diff
@@ -377,6 +409,7 @@ def main():
 
     module = AnsibleModule(argument_spec=argument_spec,
                            mutually_exclusive=[['cmdcategory', 'cmd'],
+                                               ['cmdcategory', 'cmdgroup'],
```

Comment:
So combining `cmd` with `cmdgroup` is OK?

Reply:
Yes, cmd and cmdgroup is not mutually exclusive. For example:
```
$ ipa sudorule-add-allow-command sudo_operations_all --sudocmds='/usr/bin/yum update' --sudocmdgroups='operations-cmdgroup'
  Rule name: sudo_operations_all
  Enabled: TRUE
  Sudo Allow Commands: /usr/bin/yum update
  Sudo Allow Command Groups: operations-cmdgroup
-------------------------
Number of members added 2
-------------------------
```